src/bentoml/models.py

```diff
@@ -318,6 +318,37 @@ def create(
         res.exit_cloudpickle_context(imported_modules)
 
 
+@inject
+@contextmanager
+def save(
+    name: Tag | str,
+    *,
+    module: str = "",
+    api_version: str | None = None,
```

Comment:
why expose module and api_version here?

Reply:
I was following the design here (at the bottom of https://www.notion.so/bentoml/Model-API-f020bba69482465198c7dcea8b23cc55). I suppose these two would not be too useful to users. In your opinion, should these be removed?

Reply:
removed these two parameters